contracts/mocks/MockFixed18.sol

```diff
@@ -60,6 +60,10 @@ contract MockFixed18 {
         return Fixed18Lib.div(a, b);
     }
 
+    function div(Fixed18 a, Fixed18 b, bool roundOutwards) external pure returns (Fixed18) {
```

Comment:
note: we need to do this for all of the `.mul()` functions as well since they can have the same problem while dividing by the `BASE`

Reply:
`.mul()` and the two `.mulDiv()`s, right?

Reply:
oh yes those too!